masonite/request.py

```diff
@@ -613,7 +612,8 @@ def route(self, name, params={}):
             masonite.routes.Route|None -- Returns None if the route cannot be found.
```

Comment:
Needs to specify the full=False param and what is it purpose 👍

Reply:
Also adding a convenience function with the full=True would be helpful

```python
def route_full(self, name, params={}):
  return self.route(self, name, params, full=True)
```

Reply:
good catch on the doc string. I'm not too crazy about wrapping the method in another method though